### DISCUSSION TITLE
Is there any flutter pub.dev like package repository for flet community?

### DISCUSSION THREAD

**Discussion by inghuguesboni** [flet-dev/flet#860] [General]
Hi!

I'm new in Flet. I'd like to know if there is any place where flet community shares components, widgets, ...
I mean a kind of Flutter https://pub.dev/ like version for Flet.

Thanks.

**Comment by ndonkoHenri** [comment]
https://discord.com/channels/981374556059086931/1060276348263862343
On Discord.
